New kits

katnep opened the pull request:
so this part here makes it so that fulton packs can be used on people instead of just objects(however you cannot remove yourself), removes the medevac packs because they are now redundant, and adds a bundle of survival pens for 1000 (which at this point i think might be too low so i'm gonna boost that up to at least 1500)

katnep committed:
my fulton changes

fulton rework to make them worth getting

katnep committed:
Fulton rework adjustments

fixed the spelling error and edited the description to reflect the fact
you can move people

katnep committed:
SURVIVAL BUNDLES

survival bundles which are basically just stimpack bundles but with
survival pens instead of stimpacks

katnep committed:
survival kit

raised the price to 1000

katnep committed:
few more items

ok a few things
added a survival knife to the mining vendor i don't need to tell you how
annoying it is to lose your knife. for 300 points

survival kit, containing 3 medpens, 2 survival pods, a survival knife,
and a whetstone for 1250 points

and a whetstone for 1000

katnep committed:
Removed the equipment package

it was pretty much unnecessary and probably OP when you can get a
literal spacesuit for almost the same

TalkingCactus commented on code/modules/mining/machine_vending.dm:
```diff
@@ -24,14 +25,15 @@
 		new /datum/data/mining_equipment("GAR scanners",		/obj/item/clothing/glasses/meson/gar,					  		   		500),
 		new /datum/data/mining_equipment("Explorer's Webbing",	/obj/item/weapon/storage/belt/mining,									500),
-		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			500),
+		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			400),
```
Why was this price reduced? These are way too powerful to begin with.

TalkingCactus commented on code/modules/mining/machine_vending.dm:
```diff
@@ -24,14 +25,15 @@
 		new /datum/data/mining_equipment("Resonator",           /obj/item/weapon/resonator,                                    	   		800),
-		new /datum/data/mining_equipment("Medivac Balloon",     /obj/item/weapon/extraction_pack/medivac,                               800),
+		new /datum/data/mining_equipment("whetstone",           /obj/item/weapon/sharpener,                                             1000),
+		new /datum/data/mining_equipment("Survival bundle",		/obj/item/weapon/storage/box/medipens/utility1,	 				  		1000),
```
If this is a box of survival pens, you may want to increase the price and capitalize+clarify the title. Maybe 
`Survival Medipen Bundle`

katnep committed:
medipens

medipens now cost 600 points because fuck being able to afford like 3 of
these 15 minutes in
the medipen bundle now costs 1500, and is renamed to Survival medipen
bundle

katnep committed:
making the bundle super priced

bundle price now 2000
